### PR TITLE
call https on ip-api.com/json/

### PR DESCRIPTION
**Pull request by qwerin**

### CODE REVIEW

**Comment by cemerson** [comment]
This going to happen or no? I'm running into a CSP error/block due to this - still trying to get around it. FWIW I tried setting it to HTTPS myself in a local copy but it seems to fail over SSL (https://ip-api.com/json). I'd be curious what this call is actually doing and if it can be optionally disabled from cookiebar for people like me who can't use HTTP.

**Comment by ToX82** [comment]
Unfortunately, no. ip-api.com requires a paid api key in order to be used through https, so we will be using http until we find a better solution.

If you try to use https, as in this PR, you will only get this result:
`{"status":"fail","message":"SSL unavailable for this endpoint, order a key at https://members.ip-api.com/"}`

**Comment by cemerson** [comment]
> Unfortunately, no. ip-api.com requires a paid api key in order to be used through https, so we will be using http until we find a better solution.
> 
> If you try to use https, as in this PR, you will only get this result: `{"status":"fail","message":"SSL unavailable for this endpoint, order a key at https://members.ip-api.com/"}`

Thanks @ToX82.  Can you summarize any downsides/repercussions of commenting out that json call in the core cookiebar js file?

**Comment by ToX82** [comment]
For most websites, having a `Mixed content` warning is not a big deal, so the default behavior is going to stay the same, I guess.

There is an option called `noGeoIp` that skips the geolocation call, though. By enabling that configuration, ip-api (or any other geoip service we will switch to in the future) would not be called anymore, and you won't see the error.

As an example:
`<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/cookie-bar/cookiebar-latest.min.js?always=1&noGeoIp=1"></script>`
